### ISSUE TITLE
Issue with default `style`

### ISSUE THREAD

**Issue by kavanase** [SMTG-Bham/galore#52]
Using `galore` on `python = 3.10`, `matplotlib = 3.9.0`, I get the following error when `--style` is not set:

```
 galore -g 0.1 --w he2 --flipx --xmin -7 -p galore_UPS vasprun.xml.gz
Traceback (most recent call last):
  File "/Users/kavanase/miniconda3/lib/python3.10/site-packages/matplotlib/style/core.py", line 137, in use
    style = _rc_params_in_file(style)
  File "/Users/kavanase/miniconda3/lib/python3.10/site-packages/matplotlib/__init__.py", line 870, in _rc_params_in_file
    with _open_file_or_url(fname) as fd:
  File "/Users/kavanase/miniconda3/lib/python3.10/contextlib.py", line 135, in __enter__
    return next(self.gen)
  File "/Users/kavanase/miniconda3/lib/python3.10/site-packages/matplotlib/__init__.py", line 847, in _open_file_or_url
    with open(fname, encoding='utf-8') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'seaborn-colorblind'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/kavanase/miniconda3/bin/galore", line 33, in <module>
    sys.exit(load_entry_point('galore', 'console_scripts', 'galore')())
  File "/Users/kavanase/Packages/galore/galore/cli/galore.py", line 54, in main
    run(**args)
  File "/Users/kavanase/Packages/galore/galore/cli/galore.py", line 72, in run
    simple_dos_from_files(**kwargs)
  File "/Users/kavanase/Packages/galore/galore/cli/galore.py", line 161, in simple_dos_from_files
    matplotlib.pyplot.style.use(kwargs['style'])
  File "/Users/kavanase/miniconda3/lib/python3.10/site-packages/matplotlib/style/core.py", line 139, in use
    raise OSError(
OSError: 'seaborn-colorblind' is not a valid package style, path of style file, URL of style file, or library style name (library styles are listed in `style.available`)
```

Setting `style` to a local style file, everything runs fine.

**Comment by ajjackson** [comment]
Looks like a different style name should be used for recent matplotlib versions:

https://stackoverflow.com/questions/74716259/the-seaborn-styles-shipped-by-matplotlib-are-deprecated-since-3-6

**Comment by ajjackson** [comment]
Fixed in #53
